frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -675,465 +919,376 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = ({
-        ),
-    )
-    .filter((f) => f.capabilities.includes(SpecCapability.advanced));
+  const renderFieldGroups = () =>
+    _.map(_.sortBy(fieldGroups, 'groupName'), ({ fieldList, groupName }) => {
+      if (_.isEmpty(groupName) || _.isEmpty(groupName)) {
+        return null;
+      }
+      return (
+        <div id={groupName} key={groupName}>
+          <FieldGroup
+            defaultExpand={
+              !_.some(
+                fieldList,
+                (f) => f.capabilities.includes(SpecCapability.advanced) && !f.required,
+              )
+            }
+            groupName={_.startCase(groupName)}
+          >
+            {_.map(fieldList, (field) => (
+              <OperandFormInputGroup
+                key={field.path}
+                error={formErrors?.[field.path]}
+                field={field}
+                input={inputFor(field)}
+              />
+            ))}
+          </FieldGroup>
+        </div>
+      );
+    });
 
+  const renderNormalFields = () =>
+    _.map(normalFields, (field) => (
+      <OperandFormInputGroup
+        key={field.path}
+        field={field}
+        input={inputFor(field)}
+        error={formErrors?.[field.path]}
+      />
+    ));
+
+  const renderAdvancedFields = () =>
+    advancedFields.length > 0 && (
+      <div>
+        <ExpandCollapse
+          textExpanded="Advanced Configuration"
+          textCollapsed="Advanced Configuration"
+        >
+          {_.map(advancedFields, (field) => (
+            <OperandFormInputGroup
+              key={field.path}
+              field={field}
+              input={inputFor(field)}
+              error={formErrors[field.path]}
+            />
+          ))}
+        </ExpandCollapse>
+      </div>
+    );
   useScrollToTopOnMount();
 
   return (
-    <div className="co-m-pane__body">
-      <div className="row">
-        <form className="col-md-8 col-lg-7" onSubmit={submit}>
-          <Accordion asDefinitionList={false} className="co-create-operand__accordion">
-            <div className="form-group">
-              <label className="control-label co-required" htmlFor="name">
-                Name
-              </label>
-              <input
-                className="pf-c-form-control"
-                type="text"
-                onChange={({ target }) => updateFormValues('metadata.name', target.value)}
-                value={formValues['metadata.name']}
-                id="metadata.name"
-                required
-              />
-            </div>
-            <div className="form-group">
-              <label className="control-label" htmlFor="tags-input">
-                Labels
-              </label>
-              <SelectorInput
-                onChange={(labels) => updateFormValues('metadata.labels', labels)}
-                tags={formValues['metadata.labels']}
-              />
+    <>
+      <div className="co-create-operand__header">
```

Comment:
Moved the heading stuff from CreateOperand component so that we can dispatch an update to the buffer when switching between form and operand.